source/index.rst

```diff
@@ -89,45 +83,22 @@ the chat provider, ask for its registration form and then present it to you.
 Different chat providers have different registration forms, but they're all
 relatively similar.
 
-Check the security score
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-At the top of the form you'll see a colored bar containing the text **xmpp.net score**.
-This shows the security score for this server as determined by `xmpp.net <https://xmpp.net>`_
-where you can test out a server for yourself. You should generally avoid providers with
-a poor security score (colored in red).
-
-Choosing a username
-~~~~~~~~~~~~~~~~~~~
```

Comment:
Hi @lyabs243, why did you remove the text below? Now with the newer version in your branch, there is no information on how to register an XMPP account.